appveyor.yml

```
@@ -50,7 +50,7 @@ build_script:
   - "python -c \"import awkward; print(awkward.__version__)\""
   - "pip install \"uproot-methods>=0.7.0\""
   - "python -c \"import uproot_methods; print(uproot_methods.__version__)\""
-  - "pip install cachetools lz4 mock xxhash"
+  - "pip install cachetools zstd mock xxhash"
```

I guess this is a typo:
```suggestion
  - "pip install cachetools lz4 zstd mock xxhash"
```

---

You were faster than me! I included this in my review below. (Resolving for duplication.)

Do you know about availability of the zstd Python bindings in conda? (The other comment down below.)